gateway_code/control_nodes/cn_iotlabstalone/__init__.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding:utf-8 -*-
```

Comment:
stalone looks like a typo to me, I guess the directory should be named cn_standalonem3 ?

Reply:
and why not simply `cn_iotlabm3` ?

Reply:
OK, Changed the name to `cn_iotlabm3 `.